astropy/table/pprint.py

```diff
@@ -13,19 +13,26 @@
 import numpy as np
 
 from .. import log
-# Note, in numpy <= 1.6, some classes do not properly represent themselves.
 from ..utils.console import Getch, color_print, terminal_size, conf
 from ..utils.data_info import dtype_info_name
 
-if six.PY3:
+__all__ = []
```

Comment:
I'd rather not because of this in `docs/table/index.rst`:

```
.. automodapi:: astropy.table
```

Adding something in `__all__` might change what is listed in the API doc, which is not the intend of this PR. So, either I remove it or keep it as empty list.